### PR TITLE
create fibonacci.py

### PR DESCRIPTION
**Pull request by mouar**
it prints fibonacci series

(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [ ] **If creating a new file :**
  - [ ] added links to it in the README files ?
  - [ ] included tests with it ?
  - [ ] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [ ] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

File: algorithms/maths/fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+n=int(input("enter the no of terms "))#input from the user
```
Comment (keon) [inline algorithms/maths/fibonacci.py]: please wrap the code in function

Comment (keon) [reply algorithms/maths/fibonacci.py]: and add unit tests

**Comment by keon** [comment]
Closing as stale — the repo was completely restructured in v1.0.0. Thank you for the contribution\!
